bindings/python/src/OpenSpaceToolkitAstrodynamicsPy/EventCondition.cpp

```diff
@@ -37,6 +40,113 @@ class PyEventCondition : public EventCondition
 inline void OpenSpaceToolkitAstrodynamicsPy_EventCondition(pybind11::module& aModule)
 {
     {
+        class_<EventCondition::Target> eventConditionTarget_class(
+            aModule,
+            "EventConditionTarget",
+            R"doc(
+                The Event Condition Target.
+
+                Group:
+                    event-condition
+            )doc"
+        );
+
+        enum_<EventCondition::Target::Type>(
+            eventConditionTarget_class,
+            "Type",
+            R"doc(
+                Event Condition Target type.
+            )doc"
+        )
+
+            .value("Absolute", EventCondition::Target::Type::Absolute, "Absolute")
+            .value(
+                "RelativeSegmentStart",
+                EventCondition::Target::Type::RelativeSegmentStart,
+                "Relative to the State at the end of the previous Segment."
```

Comment:
```suggestion
                "Relative to the State at the start of the current Segment."
```
nit: just because the enum is `RelativeSegmentStart` and (I think?) it's equivalent.